src/tck/Reactive.Streams.TCK.Tests/IdentityProcessorVerificationDelegationTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using NUnit.Framework;
+
+namespace Reactive.Streams.TCK.Tests
+{
+    /// <summary>
+    /// The <see cref="IdentityProcessorVerification{T}"/> must also run all tests from
+    /// <see cref="PublisherVerification{T}"/> and <see cref="SubscriberWhiteboxVerification{T}"/>.
+    /// 
+    /// Since in .Net this can be only achieved by delegating, we need to make sure we delegate to each of the tests,
+    /// so that if in the future we add more tests to these verifications we're sure to not forget to add the delegating methods.
+    /// </summary>
+    public class IdentityProcessorVerificationDelegationTest
```

Comment:
Thanks for including this test, it saved us from silly mistakes in the JVM impl back then :)